src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -1207,14 +1207,27 @@ public virtual void EnsureBackupDirectoryRemoved(string packageName)
                                     null,
                                     null));
 
+                        // For an initial attempt at finding a package resolution solution, check for all parent packages (i.e. locally installed packages

```

Comment:
aside: I still think we should rename the whole "parent" package thing to "dependent" packages, "parent" doesn't make much sense :/